profile/Makefile

```diff
@@ -13,7 +13,7 @@ PRECISION   = single
 # Source Code List
 #===============================================================================
 
-case = c5g7/c5g7-cmfd.cpp
+case = c5g7/c5g7-cmfd.cpp 
```

Comment:
Remove the space after ``c5g7/c5g7-cmfd.cpp``. Per our style guide, all trailing white spaces should be removed.